presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/Memo.java

```diff
@@ -118,35 +122,34 @@ public PlanNode replace(int group, PlanNode node, String reason)
             node = insertChildrenAndRewrite(node);
         }
 
-        incrementReferenceCounts(node);
+        incrementReferenceCounts(node, group);
         getGroup(group).membership = node;
-        decrementReferenceCounts(old);
+        decrementReferenceCounts(old, group);
 
         return node;
     }
 
-    private void incrementReferenceCounts(PlanNode node)
+    private void incrementReferenceCounts(PlanNode fromNode, int fromGroup)
```

Comment:
why not jus `node` and `group` as it used to be?

Reply:
Because `group` is used as a variable inside method body